Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/nhinclib/NhincConstants.java

```diff
@@ -211,19 +211,12 @@ public static EVENT_LOGGING_SERVICE_NAME fromValueString(String valueString) {
 
     // Authorization Framework
     public static final String AUTH_FRWK_NAME_ID_FORMAT_EMAIL_ADDRESS
-        = "urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress";
+    = "urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress";
```

Comment:
Should this move as well?  Anything in // Authorization Framework and AUTHZ_DECISION_STMT_ACTION_NAMESPACE